spec/articleListSpec.js

```diff
@@ -0,0 +1,16 @@
+var articleListModule = require('../src/articleList.js')
+
+describe('ArticleList', function() {
+  var articleList;
+
+  beforeEach(function(){
+    articleList = new articleListModule.ArticleList();
+  });
+
+  describe('#articles', function(){
+    it('has an empty array', function(){
+        expect(articleList.articles()).toBeEmpty();
```

Comment:
State not behaviour!

As an exercise, try laying your tests out in the format:

```
// Arrange
... prepare your test, e.g. create the test instance, mock dependencies etc

// Act
... perform the action that you are trying to test, which should be a method on the instance being tested

// Assert
... your expect(...) statement(s)

```
If you lay your tests out like this and you find that (a) there is no code in the 'Act' section or (b) your test would pass if you commented out the 'Act' section then the chances are that you are testing state rather than behaviour.

This applies to some other tests as well. I dropped a link to a short article into the October channel as this is something that I see in quite a lot of code reviews.